setup/azurecmdfiles.wxi

```diff
@@ -4,6 +4,62 @@
         <DirectoryRef Id="PowerShellFolder">
             <Directory Id="dirB60B5B1F56FA5598EFFC960CA393FDCE" Name="ResourceManager">
                 <Directory Id="dir9E409EB63EF03736275B51044C819FD8" Name="AzureResourceManager">
+                    <Directory Id="dirE5EE64AF112D660B2F38178FA55F7183" Name="Azure.AnalysisServices">
```

Comment:
@hovsepm the entire regen need to happen because I removed .js and .json files from the two remaining RPs that still had them. And caused the entire regen.